.github/workflows/merge-queue.yml

```
@@ -3,8 +3,103 @@ name: Validate code in the merge queue (post merge)
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          enable-cache: 'node_modules,cypress,generated-files'
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+
+      - name: Setup yarn
+        run: corepack enable
+      - name: Gather branch info
+        id: branch_info
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:99: Double quote to prevent globbing and word splitting

(shellcheck)